### PR TITLE
docs: add offline-installation.md

### PR DESCRIPTION
**Pull request by Joyinqin**
Signed-off-by: Joyinqin <joyinqin@pku.edu.cn>
This pr is translated from: https://github.com/chaos-mesh/website-zh/pull/15

### CODE REVIEW

File: docs/offline-installation.md [hunk 1]
```diff
@@ -1,6 +1,157 @@
...
+## Prerequisites
+
+Before installing Chaos Mesh, make sure that Docker is installed and the Kubernetes cluster is deployed in the offline environment. If the environment is not prepared, refer to the following documents to install Docker and deploy the Kubernetes cluster:
+
+- [Docker](https://www.docker.com/get-started)
+- [Kubernetes](https://kubernetes.io/docs/setup/)
+
+## Prepare installation files
+
+Before you install Chaos Mosh offline, you need to download all Chaos Mesh images and repository compressed packages from machines with external network connections, and then copy the downloaded files into your offline environment.
```
Comment (TomShawn) [inline docs/offline-installation.md:22]:
```suggestion
Before you install Chaos Mesh offline, you need to download all Chaos Mesh images and repository compression packages from the machines with external network connection, and then copy the downloaded files into your offline environment.
```

File: docs/offline-installation.md [hunk 2]
```diff
@@ -1,6 +1,157 @@
...
+
+Set the version number of Chaos Mesh as the environment variable on the machine with external network connection:
+
+<PickVersion className="language-bash">
+export CHAOS_MESH_VERSION=latest
+</PickVersion>
+
+### Download Chaos Mesh images
+
+On the machine connected to the external network, pull images using the version number that has been set:
```
Comment (TomShawn) [inline docs/offline-installation.md:34]:
```suggestion
On the machine connected to external network, pull images using the version number that has been set:
```

File: docs/offline-installation.md [hunk 3]
```diff
@@ -1,6 +1,157 @@
...
+:::note
+
+Before pushing images to Registry, make sure that Registry has been deployed in the offline environment. If Registry is not deployed, refer to [Docker Registry](https://docs.docker.com/registry/) for the deployment method.
+
+:::
+
+Set the Chaos Mesh version and the Registry address as the environment variable:
+
+<PickVersion className="language-bash">
+export CHAOS_MESH_VERSION=; export DOCKER_REGISTRY=localhost:5000
```
Comment (TomShawn) [inline docs/offline-installation.md:106]:
```suggestion
export CHAOS_MESH_VERSION=latest; export DOCKER_REGISTRY=localhost:5000
```

File: docs/offline-installation.md [hunk 4]
```diff
@@ -1,6 +1,157 @@
...
+unzip chaos-mesh.zip -d chaos-mesh && cd chaos-mesh
+```
+
+Create the namespace:
+
+```bash
+kubectl create ns chaos-testing
+```
+
+Execute the installation command.  When executing the installation command, you need to specify the namespace of Chaos Mesh and the image value of each component:
```
Comment (TomShawn) [inline docs/offline-installation.md:142]:
```suggestion
Execute the installation command. When executing the installation command, you need to specify the namespace of Chaos Mesh and the image value of each component:
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: docs/offline-installation.md [hunk 5]
```diff
@@ -1,6 +1,157 @@
...
+
+```bash
+kubectl create ns chaos-testing
+```
+
+Execute the installation command. When executing the installation command, you need to specify the namespace of Chaos Mesh and the image value of each component:
+
+```bash
+helm install chaos-mesh helm/chaos-mesh -n=chaos-testing \
+  --set chaosDaemon.image=$CHAOS_DAEMON_IMAGE
```
Comment (g1eny0ung) [inline docs/offline-installation.md:146]: Miss a backslash at the end.

Comment (TomShawn) [reply docs/offline-installation.md:146]:
```suggestion
  --set chaosDaemon.image=$CHAOS_DAEMON_IMAGE \
```

**Comment by TomShawn** [comment]
@g1eny0ung PTAL again. thanks!

**Pull request merged**